Gruntfile.js

```diff
@@ -21,7 +21,7 @@ module.exports = function (grunt) {
         options: {
           html: 'public/index.html'
         },
-        src: 'design/icon.png',
+        src: 'app/assets/web-app-icon.png',
```

Comment:
This breaks favicon generation.  `icon.16x16.png` is no longer used for the generated `favicon.ico`.

That said, comparing `icon.16x16.png` and the generated imagemagick icon, I think the quality of the latter is good enough.  So can you delete `icon.16x16.png` please?